rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -255,6 +271,19 @@ def test_exec(config):
                         if config.local_file_delete is True:
                             log.info("deleting local file created after the upload")
                             utils.exec_shell_cmd("rm -rf %s" % s3_object_path)
+                    if config.dynamic_resharding is True:
+                        reusable.check_sync_status()
+                        time.sleep(600)  # Adding delay for processing reshard list
```

Comment:
This is very high... and adds time to the execution. Is there a way we can check if the reshard processing is done?

Reply:
Since we are not sure that how much time the current resharding process will take, We are setting this time little more

Reply:
Added for loop to deal with sleep time